### PR TITLE
Customize help. Sync options: testrail, azuredevops.

### PR DESCRIPTION
**Pull request by Brrovko**
- Customize help
- Add sync options: testrail, azuredevops
- README.md updated

### CODE REVIEW

File: GherkinSyncTool/CommandLineOptions/CommandLineParserHelper.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+using CommandLine;
+using CommandLine.Text;
+using NLog;
+
+namespace GherkinSyncTool.CommandLineOptions
+{
+    public static class CommandLineParserHelper
+    {
+        private static readonly Logger Log = LogManager.GetLogger(MethodBase.GetCurrentMethod()?.DeclaringType?.Name);
+        private static readonly string Heading = $"{Program.Version}Quantori GherkinSyncTool is an open-source console " +
```
Comment (vladimir-s-quantory) [inline GherkinSyncTool/CommandLineOptions/CommandLineParserHelper.cs:14]: Let's use interpolation?

Comment (ILastovka) [inline GherkinSyncTool/CommandLineOptions/CommandLineParserHelper.cs:14]:
```suggestion
        private static readonly string Heading = $"{Program.Version} Quantori GherkinSyncTool is an open-source console " +
```

**Review by ILastovka — CHANGES_REQUESTED** [review]

File: GherkinSyncTool/Program.cs [hunk 2]
```diff
@@ -74,11 +72,33 @@ private static int Main(string[] args)
...
+            parserResult
+                .WithParsed(RegisterSynchronizer)
+                .WithNotParsed(errs => CommandLineParserHelper.DisplayHelp(parserResult, errs));
+        }
+
+        private static void RegisterSynchronizer(object options)
         {
+            var syncOptions = (SyncOptions) options;
+
+            if (syncOptions.Testrail)
```
Comment (vladimir-s-quantory) [inline GherkinSyncTool/Program.cs:88]: I guess we better use switch statement here if we imply several test environments

File: GherkinSyncTool/CommandLineOptions/CommandLineParserHelper.cs (hunk 1 above)
Comment (Brrovko) [reply GherkinSyncTool/CommandLineOptions/CommandLineParserHelper.cs:14]: `Program.Version` has a line brake, so it looks ok.
image

**Review by SergeyRuzin — APPROVED** [review]

**Review by ILastovka — APPROVED** [review]

File: GherkinSyncTool/CommandLineOptions/CommandLineParserHelper.cs (hunk 1 above)
Comment (Brrovko) [reply GherkinSyncTool/CommandLineOptions/CommandLineParserHelper.cs]: Done

File: GherkinSyncTool/Program.cs (hunk 2 above)
Comment (Brrovko) [reply GherkinSyncTool/Program.cs]: Done

**Review by vladimir-s-quantory — APPROVED** [review]

**Pull request merged**